source/slang/hlsl.meta.slang

```diff
@@ -1117,78 +1290,103 @@ vector<TElement,4> __glsl_gatherCmp_offsets(__TextureImpl<T, Shape, isArray, 0,
         };
     }
 }
+__generic<TElement, T, Shape: __ITextureShape, let isArray:int, let sampleCount:int, let access:int, let isShadow:int, let format:int>
+[ForceInline]
+vector<TElement,4> __glsl_gatherCmp_offsets(__TextureImpl<T, Shape, isArray, 0, sampleCount, access, isShadow, 1, format> sampler, vector<float, Shape.dimensions+isArray> location, TElement compareValue,
```

Comment:
Are these additional `__glsl` helper functions used by other stuff in hlsl.meta.slang? If not we can move them to glsl.meta.slang.